### PR TITLE
LPS-184638 Add a checkbox to set/unset the propagation of permissions when creating a new folder

### PR DESCRIPTION
**Pull request by ces-luannguyen**
### Motivation
This work implements features reported in the [LPS-184638](https://issues.liferay.com/browse/LPS-184638) ticket (as part of [LPS-154515](https://issues.liferay.com/browse/LPS-154515) ticket).
CC: @locpham97
**1. Requirement:**
Currently, after creating a folder, if a document manager wants to configure its permission, he/she will do it in permission modal. This ticket is for propagating permissions from parent folder to subfolder (child folder) automatically without configuring in permission modal if permission propagation checkbox is selected. This checkbox is shown in folder edit form when user creates a new folder.
**2. What we did to implement this feature:**
There's a permission section in folder edit form using `liferay-ui:input-permissions` tag lib. We custom this tag lib `liferay-ui:input-permissions` by adding additional `documentMedia` to this tag lib attribute. If this attribute is true, permission propagation checkbox is shown for selecting, otherwise this checkbox is not shown in folder edit form.

### CODE REVIEW

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -15052,6 +15052,7 @@ set-expiration-date=Set Expiration Date
 set-filters=Set Filters
 set-parent-price-list=Set Parent Price List
 set-password=Set Password
+set-permissions-for-documents-and-folders-uploaded-to-this-folder=Set permissions for documents and folders uploaded to this folder.
```
Comment (locpham97) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:15055]: Build lang for this language key

**Comment by ces-luannguyen** [comment]
I fixed code review. Please help me double check. @locpham97 

Thanks,
Luan Nguyen

File: util-taglib/src/com/liferay/taglib/ui/InputPermissionsTag.java [hunk 2]
```diff
@@ -30,20 +30,24 @@
 public class InputPermissionsTag extends IncludeTag {
 
 	public static String doTag(
-			String formName, String modelName, PageContext pageContext)
+			String formName, String modelName, boolean documentMedia,
```
Comment (locpham97) [inline util-taglib/src/com/liferay/taglib/ui/InputPermissionsTag.java:33]: should sort this args

**Review by ces-locpham — APPROVED** [review]

**Comment by ces-locpham** [comment]
Hi @tototrinh ,
This is passed my review. Could you check this PR?
Thanks,
Loc

**Comment by ces-totrinh** [comment]
Resend here: https://github.com/tototrinh/liferay-portal/pull/53

**Pull request closed without merging**